spec/rails_helper.rb

```diff
@@ -20,19 +20,20 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join("spec", "support", "**", "*.rb")].sort.each { |f| require f }
+# Rails.root.glob('spec/support/**/*.rb').sort.each { |f| require f }
```

Comment:
We technically still need this line.
```suggestion
  Rails.root.glob('spec/support/**/*.rb').sort.each { |f| require f }
```